src/main/java/org/spongepowered/common/event/lifecycle/RegisterDataPackValueEventImpl.java

```diff
@@ -43,6 +46,8 @@
     public RegisterDataPackValueEventImpl(final Cause cause, final Game game) {
         super(cause, game);
         this.serializables = new Object2ObjectOpenHashMap<>();
+        this.serializables.computeIfAbsent((SpongeDataPackType) DataPackTypes.RECIPE, k -> new ArrayList<>());
```

Comment:
Shouldn't this be fixed in a getter call? Would rather not hard code like this especially when more datapack things are added...

Reply:
Its so that both datapacks are actually deleted when no plugin registers anything for it.
There is no getter call for a specific DataPackType its just `event.getSerializables()` but I'll move it to `SpongeDataPackManager`